### PR TITLE
Update Cortex versions (eksctl, EKS, AWS IAM, Python, etc)

### PR DESCRIPTION
**Pull request by RobertLucian**
This PR updates Cortex's versions of everything as instructed in the `versions.md` README. It also adds the latest instances AWS has started supporting, like the `g5` instances.

---

checklist:

- [x] run `make test` and `make lint`
- [x] test manually (i.e. build/push all images, restart operator, and re-deploy APIs)
- [x] update docs and add any new files to `summary.md` (view in [gitbook](https://cortex-labs.gitbook.io/staging/-MOmCGMADSRNQahK3Kox/) after merging)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -37,11 +37,11 @@ require (
...
-	istio.io/api v0.0.0-20210617170320-ee3eb4b39496
-	istio.io/client-go v1.10.2-0.20210617171818-3dcf18fc084e
-	k8s.io/api v0.20.8
-	k8s.io/apimachinery v0.20.8
-	k8s.io/client-go v0.20.8
+	istio.io/api v0.0.0-20220304045306-249321c725a9
+	istio.io/client-go v1.10.6
+	k8s.io/api v0.20.15
+	k8s.io/apimachinery v0.20.15
+	k8s.io/client-go v0.20.15
```
[the remark this answers is not in the document]
Comment (RobertLucian) [reply go.mod:44]: I see. Addressing the comment now.

[the remark this answers is not in the document]
Comment (RobertLucian) [reply go.mod:44]: @miguelvr should we also update `k8s.io/metrics`?

[the remark this answers is not in the document]
Comment (RobertLucian) [reply go.mod]: Tried upgrading the `k8s.io/metrics` package, but I got into the same error we got before: https://app.circleci.com/pipelines/github/cortexlabs/cortex/11888/workflows/ee654403-a4a6-4edf-8f6b-3d4285140ecf/jobs/16340. Gonna revert it back.

[the remark this answers is not in the document]
Comment (RobertLucian) [reply go.mod]: Fixed this now.

**Review by deliahu — APPROVED** [review]

**Pull request merged**